packages/data-service/src/data-service.ts

```diff
@@ -1381,6 +1389,11 @@ class DataServiceImpl extends WithLogContext implements DataService {
         this._tunnel
           ?.close()
           .catch((err) => debug('failed to close tunnel', err)),
+        this._state
+          ?.destroy()
+          .catch((err) =>
+            debug('failed to destryo DevtoolsConnectionState', err)
```

Comment:
Sorry for typo corrections - I now have this vscode extension that points them all out to me and I can't unsee 😅 
```suggestion
            debug('failed to destroy DevtoolsConnectionState', err)
```